fix: ignore warning binding non reactive (#12524)
sveltejs/kit: .changeset/gorgeous-jars-sparkle.md

---
'@sveltejs/kit': patch
---

> fix: ignore binding_non_reactive in generated root component (this will cause an extra warning in `svelte@5.0.0-next.204`)

Reviewer: ```suggestion
fix: ignore `binding_non_reactive` warning in generated root component (you also need to update to `svelte@5.0.0-next.204`)
```

> fix: ignore `binding_non_reactive` warning in generated root component (you also need to update to `svelte@5.0.0-next.204`)

Reviewer: ```suggestion
fix: Svelte 5 - ignore `binding_non_reactive` warning in generated root component (you also need to update to `svelte@5.0.0-next.204`)
```
